pom.xml

```diff
@@ -3202,185 +3177,6 @@
-                                            <report>modules</report>
-                                            <report>plugin-management</report>
-                                            <report>plugins</report>
-                                            <report>project-team</report>
-                                            <report>scm</report>
-                                            <report>summary</report>
-                                        </reports>
-                                    </plugin>
-                                    <plugin>
-                                        <groupId>org.apache.maven.plugins</groupId>
-                                        <artifactId>maven-jxr-plugin</artifactId>
-                                        <configuration>
-                                            <aggregate>true</aggregate>
-                                        </configuration>
-                                    </plugin>
-                                    <plugin>
-                                        <groupId>org.apache.maven.plugins</groupId>
-                                        <artifactId>maven-dependency-plugin</artifactId>
-                                    </plugin>
-                                    <plugin>
-                                        <groupId>org.apache.maven.plugins</groupId>
-                                        <artifactId>maven-surefire-report-plugin</artifactId>
-                                        <configuration>
-                                            <aggregate>true</aggregate>
-                                        </configuration>
-                                    </plugin>
-                                    <plugin>
-                                        <groupId>org.codehaus.mojo</groupId>
-                                        <artifactId>cobertura-maven-plugin</artifactId>
-                                        <configuration>
-                                            <aggregate>true</aggregate>
-                                            <!--
-                                                Whether to remove GPL licensed files from the generated report. This is required to distribute
-                                                the report as part of a distribution, which is licensed under the ASL, or a similar license,
-                                                which is incompatible with the GPL
-                                            -->
-                                            <omitGplFiles>true</omitGplFiles>
-                                        </configuration>
-                                    </plugin>
-                                    <plugin>
-                                        <groupId>org.apache.rat</groupId>
-                                        <artifactId>apache-rat-plugin</artifactId>
-                                        <configuration>
```

Comment:
why did you removed most configuration ?

Reply:
Configuration is not allowed in the reporting section.